### ISSUE TITLE
fps limit none feels like v-sync and causes input lag

### ISSUE THREAD

**Issue by MrKarpuz** [haxball/haxball-issues#476]
Hi basro ! I'm new here and want to explain the problem I have while playing haxball. As you know when we open v-sync in most games fps is stuck at 60 and the display is so good when turning around, but causes input lag. I'm having the same problem in haxball, the ball is uncontrollable and heavy when I hit fps limit none, it feels like my movements are delayed for miliseconds. On the other hand, when I limit 30 fps, It is so fucking smooth I can do everything with the ball but I will be blind one day because 30 fps is terrible. Tried using Chrome, Opera, Firefox nothing helped. I'm using a gaming laptop, still couldn't found the problem. I actually think fps limit none is a lie, it limits 60 like v-sync.

**Comment by Leo-MapCreator** [comment]
There is a command named extrapolation that helps to eliminated small delay inputs

**Comment by MrKarpuz** [comment]
Yes I have tried it but still wasn't smooth enough as it is on 30 fps :(
